components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/servlet/OIDCLogoutServlet.java

```diff
@@ -973,4 +975,16 @@ private void clearTokenBindingElements(String clientId, HttpServletRequest reque
         tokenBinders.stream().filter(t -> oAuthAppDO.getTokenBindingType().equals(t.getBindingType())).findAny()
                 .ifPresent(t -> t.clearTokenBindingElements(request, response));
     }
+
+    private String resolveTenantDomain(HttpServletRequest request) {
```

Comment:
So the idea hear is if the tenanted sessions are not enabled, cache will be in the super tenant?

Reply:
Yes. If the tenanted sessions are not enabled, cache will be added to the super tenant.